### DISCUSSION TITLE
map/surf_salmari

### DISCUSSION THREAD

**Discussion by giscus** [Kalekki/SurfHeaven_Extended#9] [Announcements]
# map/surf_salmari

Click the download icon to download surf_salmari. You can also see the top 100 records for the map as well as various other information.

https://surfheaven.eu/map/surf_salmari

**Comment by Kalekki** [comment]
Best map of the season ![surfing monkey :3](https://cdn.7tv.app/emote/625e903d62fe096141e61a33/4x.webp)
